code/modules/urist/datums/xenoarch_effects/poltergeist.dm

```diff
@@ -0,0 +1,42 @@
+/datum/artifact_effect/poltergeist
+	name = "poltergeist"
+	effect_type = EFFECT_ENERGY
+
+
+/datum/artifact_effect/poltergeist/DoEffectTouch(mob/living/user)
+	throw_at_mob(user, rand(10, 30))
+
+
+/datum/artifact_effect/poltergeist/DoEffectAura()
+	var/mob/living/target = null
+	for (var/mob/living/L in oview(get_turf(holder), effectrange))
+		if (L.stat || !L.mind)
+			continue
+		if (target && get_dist(get_turf(holder), L) > get_dist(get_turf(holder), target))
+			continue
+		target = L
+	if (target)
+		throw_at_mob(target, rand(15, 30))
+
+
+/datum/artifact_effect/poltergeist/DoEffectPulse()
+	var/mob/living/target = null
+	for (var/mob/living/L in oview(get_turf(holder), effectrange))
+		if (L.stat || !L.mind)
+			continue
+		if (target && get_dist(get_turf(holder), L) > get_dist(get_turf(holder), target))
+			continue
+		target = L
+	if (target)
+		throw_at_mob(target, chargelevelmax)
+
+
+/datum/artifact_effect/poltergeist/proc/throw_at_mob(mob/living/target, damage = 20)
+	var/list/valid_targets = list()
+	for (var/obj/O in oview(world.view, target))
+		if (!O.anchored && isturf(O.loc))
+			valid_targets |= O
+	if (length(valid_targets))
+		var/obj/obj_to_throw = pick(valid_targets)
+		obj_to_throw.visible_message("<span class='alien'>\The [obj_to_throw] levitates, before hurtling toward [target]!</span>")
+		obj_to_throw.throw_at(target, world.view, min(40, damage * GetAnomalySusceptibility(target)))
```

Comment:
A bit weird that the damage scales by the anomaly susceptibility of what we're throwing stuff *at* rather than the thing we throw. 

In principle, an unshielded guy standing right behind an anomaly-shielded guy could somehow cause the latter to take twice as much damage when acting as a human shield than when the shielded guy gets the item yeeted at the him directly.

Reply:
Now adjusted to scale by w_class